back/cmd/openbsrv/go.mod

```diff
@@ -9,7 +9,8 @@ require (
 	github.com/codemodus/hedrs v0.1.1
 	github.com/codemodus/sigmon/v2 v2.0.1
 	github.com/codemodus/sqlmig v0.2.3
-	github.com/go-sql-driver/mysql v1.4.1
+	github.com/go-sql-driver/mysql v1.5.0
+	github.com/jmoiron/sqlx v1.3.1 // indirect
```

Comment:
I removed the `// indirect` part of the line, ran `go mod tidy`, and the `// indirect` came back. It doesn't make sense to me why it's there given it sqlx is a direct dependency.

Reply:
The reason for the `// indirect` comment is because it is indirect. @daved realized that this main package is importing a library that is itself dependent on sqlx.